packages/serde_debug/serde_debug.0.0.1/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis:
+  "A human-friendly format for Serde that helps you debug any data during development"
+maintainer: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+authors: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+license: "MIT"
+homepage: "https://github.com/leostera/serde.ml"
+bug-reports: "https://github.com/leostera/serde.ml/issues"
+depends: [
+  "ocaml" {>= "4.12.0"}
+  "dune" {>= "3.5" & >= "3.5"}
+  "serde"
```

Comment:
Generally, when releasing a set of package that work with one another, we use the `{ = version }` constraint to force packages to be installed at the same version.

If you follow a specific versioning scheme which ensures that your separate but related packages can work with certain version ranges of each other, then it's fine to use different constraints (e.g., `{ < "0.1" }` for "same major.minor but different point").

Without constraints, you'll likely need to modify these packages when you make a release in the future.

Reply:
Note: this comment also applies to other cross-packages dependencies.

Reply:
So I don't want to deal with dependency ranges right now, or backwards compatibility, until a v1 comes about. Would you recommend I just fix the version of everything? For ex: `(serde (= "0.0.1"))` and when I publish `0.0.2` then just move all the pointers there?

I wouldn't want a `(< "1.0")` that would let you use `0.0.1` and `0.0.2` together, because I don't want to worry about supporting those cases.

Wdyt?

Reply:
I've fixed them to `0.0.1` for now.

Reply:
I meant `{ = version }` literally. As in `version` is a keyword recognised by opam meaning "the same version as this package".
See comment by Kate below.